### DISCUSSION TITLE
Recovery my old Account.

### DISCUSSION THREAD

**Discussion by goldliondevx** [community/community#111025] [Other Feature Feedback, Questions, & Ideas]
I'm currently facing difficulties with the recovery of my GitHub account due to the inadvertent use of multiple free accounts on my device. Consequently, my main GitHub account has been suspended, as it appears I violated GitHub's policies on the usage of free accounts. Unfortunately, due to my busy schedule, I wasn't able to address this issue promptly.

Realizing the gravity of the situation, I've made payment for my current GitHub account and promptly reached out to GitHub Support for assistance in resolving the suspension. However, despite my efforts, I haven't received a response from them for over a month.

I'm reaching out to the community in the hope that someone may have insight or connections that could expedite the recovery process of my account. It's critical for me to regain access to my account as soon as possible, as it contains private projects that I urgently need to access and work on.

For reference, my support ticket number is [2596849](https://support.github.com/ticket/personal/0/2596849), and my old account can be found at [goldliondev](https://github.com/goldliondev).

I understand the importance of adhering to GitHub's terms of service and will ensure compliance moving forward. In the meantime, any assistance or guidance from fellow community members would be greatly appreciated in resolving this matter swiftly.

Thank you for your understanding and support.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket - New Ticket - Finally describe it

Good Luck!

**Reply by goldliondevx** [reply]
Thanks for your reply. @gil-son 
I requested but it become closed without any result automaticly

**Reply by gil-son** [reply]
Hum, maybe you have to wait a staff

**Comment by goldliondevx** [comment]
Any help from Github Staff? I am still facing this fuck problem.
